htdocs/core/modules/modFacture.class.php

```diff
@@ -183,6 +183,13 @@ function __construct($db)
 		$this->rights[$r][4] = 'facture';
 		$this->rights[$r][5] = 'export';
 
+		$r++;
+		$this->rights[$r][0] = 1322;
+		$this->rights[$r][1] = 'Rouvrir une facture totalement réglée';
+		$this->rights[$r][2] = 'r';
+		$this->rights[$r][3] = 0;
+		$this->rights[$r][4] = 'reopen';
```

Comment:
The default policy about right is that right to go back is same than right to to the action.
Adding right that differs from that must be done using advanced permission.
So $this->rights[$r][4] = 'reopen'; should be $this->rights[$r][4] = 'reopen_advance';